examples/wifi-echo/server/esp32/main/RendezvousDeviceDelegate.cpp

```diff
@@ -0,0 +1,102 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include "RendezvousDeviceDelegate.h"
+#include "BluetoothWidget.h"
+#include "RendezvousMessageHandler.h"
+#include "esp_log.h"
+#include <platform/ConfigurationManager.h>
+#include <support/CodeUtils.h>
+#include <support/ErrorStr.h>
+#include <system/SystemPacketBuffer.h>
+
+extern BluetoothWidget bluetoothLED;
+extern NodeId kLocalNodeId;
+extern void PairingComplete(SecurePairingSession * pairing);
+
+static const char * TAG = "rendezvous-devicedelegate";
+
+using namespace ::chip;
+
+RendezvousDeviceDelegate::RendezvousDeviceDelegate()
+{
+    CHIP_ERROR err = CHIP_NO_ERROR;
+
+    uint32_t setupPINCode;
+    err = DeviceLayer::ConfigurationMgr().GetSetupPinCode(setupPINCode);
+    SuccessOrExit(err);
+
+    mRendezvousSession = new RendezvousSession(this, RendezvousParameters(setupPINCode).SetLocalNodeId(kLocalNodeId));
+    err                = mRendezvousSession->Init();
+    SuccessOrExit(err);
```

Comment:
this is a no-op because the very next instruction is exit: ?  I'm ok with this for uniformity's sake, but would you also do this at line 66 in such case?

Reply:
I like to do it at the end in case someone is adding code after it. But you're right that I need to stay consistent. Will fix this one or line 66.

Reply:
Fixed.